.github/workflows/unittest.yml

```
@@ -0,0 +1,29 @@
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.fork == true
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.11" ]
+    steps:
+    - uses: actions/checkout@v4
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)